librz/core/cbin.c

```diff
@@ -1854,6 +1854,9 @@ RZ_API void rz_core_bin_print_source_line_info(RzCore *core, const RzBinSourceLi
 }
 
 static const char *bin_reloc_type_name(RzBinReloc *reloc) {
+	if (reloc->additive == 2) { // if we can use '.print_name'.
+		return reloc->print_name;
+	}
```

Comment:
Please use another indicator then `additive == 2`. `additive` has no relation to the name printing and it gets confusing. Couldn't you just check if `print_name` is `NULL`, or am I missing something?

Reply:
You could check if `print_name` is `NULL`, it's just adding `print_name = NULL` on every other format now. I thought one line changing `additive` would be better. It's adding a single line to the `ELF` processing. 
We will be deleting the line either way soon so I didn't think it mattered much, but I could change it

Reply:
The `reloc` struct should be initilized with `RZ_NEW0` before. So it should be already 0 by default.

Reply:
Either way I'll change the naming

Reply:
> The `reloc` struct should be initilized with `RZ_NEW0` before. So it should be already 0 by default.

Oh OK, I'll change it then

Reply:
i don't know why 2 is there, but maybe define a macro which can be used to understand why the constant 2 is there.

Reply:
> Either way I'll change the naming

Ended up not changing the name since when used in the old way, it makes sense. Added a comment though to clarify what it is.